Added basic skins

CriMsOnN opened the pull request:
Its a WIP.

Maybe you should split skins and clothes to 2 resources that would be easier to handle. 
I added basic functionality for skins
Skins included
Eyes,Heads,Hair,Height,Skin Color for now
Changed the natives for blips
Changed the control hex
(The apply command its a test command since we dont save the skins so we can load them)

BerkieBb commented:
Great tip for you, use `onResourceStart` instead of `onClientResourceStart` for putting a player logged in, otherwise it will cause problems if you leave it in a running server, onResourceStart only runs when the resource is (re)started and not when a client joins the server so it will prevent the problems

CriMsOnN commented:
You are right i changed it.
Also added all face features (37 total)
Changed the design a bit ( i am bad at designing but i gave my best)

https://imgur.com/9OZLyIQ
https://imgur.com/g59XKbC
